lib/pxapi.js

```diff
@@ -150,6 +150,11 @@ function buildRequestData(ctx, config) {
     if (ctx.hmac) {
         data.additional['px_cookie_hmac'] = ctx.hmac;
     }
+
+    data.additional['start_enforcer_timestamp'] = ctx.startEnforcerTime;
+    ctx.sendingRiskTimestamp = Date.now();
+    data.additional['sending_risk_timestamp'] = ctx.sendingRiskTimestamp;
```

Comment:
Why not call it "timestamp" and put it at the root just like async activities? Will this cause issues with the collector?

Reply:
We should send this field on async activities too. I changed it to `risk_start_time` and `enforcer_start_time`.